.circleci/src/workflows/commit.yml

```diff
@@ -15,6 +15,20 @@ jobs:
         - mattermost
       requires:
         - bump_version
+  - build_web:
```

Comment:
Why do we specity the workflow here in commit.yml and also is config.yml? Is this duplication necessary?

Reply:
`commit.yml` is autogenerated from all files in `.circleci/src`. You should never modify `commit.yml` directly therefore.
The split in multiple files makes it way clearer instead of having one big config file.